script/start-web.sh

```diff
@@ -1,12 +1,12 @@
 #! /bin/bash
 set -ue
 
-if [[ "${USE_FASTBOOT:-0}" = 1 ]]; then
-    export USE_FASTBOOT=1
+if [[ -z "${USE_FASTBOOT}" ]]; then
```

Comment:
I think [script/ember.sh](https://github.com/rust-lang/crates.io/blob/5749abaff535fc961edca47f789c933dd57707e6/script/ember.sh#L6) should be updated to match the changes here?

Reply:
> I think script/ember.sh should be updated to match the changes here?

Yeah, that makes sense.  I've updated the test there to match.